starter-code/scripts/article.js

```diff
@@ -17,10 +17,13 @@ Article.prototype.toHtml = function() {
   //   or say "(draft)" if it has no publication date:
   this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
   this.publishStatus = this.publishedOn ? 'published ' + this.daysAgo + ' days ago' : '(draft)';
-  // TODO: Use handlebars to render your articles!
+  //  : Use handlebars to render your articles!
   //       - Select your template from the DOM.
   //       - Now "compile" your template with Handlebars.
   //       - Don't forget to return your template for this article.
+  var source = $('#article-template').html();
+  var template = Handlebars.compile(source);
+  return template(this);
```

Comment:
Remember that `template` is just a custom function that the `Handlebars.compile` method returns to us, based on the html source template we specify. 👍